source/geometry/index.js

```diff
@@ -8,6 +8,29 @@ export function circlePoints(n, r) {
     return Array(n).fill(1).map((_, i) => toCircle(2 * Math.PI * i/n, r));
 }
 
+/*archimedeanSpiralPointSet
+a => the rotational offset
+b => controls the tightness and parity of the spiral
+from: https://en.wikipedia.org/wiki/Archimedean_spiral
+"in polar coordinates(r, theta) 
+r = a + b*theta
+nCycles => number of cycles
+nPoints => number of sample points in the set
+##FUTURE## even, growth, random spacing of points option
+
+*/
+function archSpiralPointSet(a, b, nCycles, nPoints){
```

Comment:
This and circlePoints should have a consistent naming convention. I'm inclined to say nix the "Set" since these are ordered arrays and not strictly speaking sets.